examples/v1alpha2/cross-namespace-routing/gateway.yaml

```diff
@@ -0,0 +1,17 @@
+kind: Gateway
+apiVersion: gateway.networking.k8s.io/v1alpha2
+metadata:
+  name: shared-gateway
+  namespace: infra-ns
+spec:
+  gatewayClassName: shared-gateway-class
+  listeners:
+  - name: http
+    hostname: "foo.example.com"
+    protocol: HTTP
+    port: 80
+    routes:
+      namespaces:
+        from: "All"
+      kinds:
+      - kind: HTTPRoute
```

Comment:
I think it would be simpler to leave this out since it's not required.

Reply:
done in 890861739ef42a42882a1241b3ae005cd833c301